model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -87,7 +87,7 @@ case class WorkspaceRequest (
 case class Workspace(
                       namespace: String,
                       name: String,
-                      authorizationDomain: Option[ManagedGroupRef],
+                      authorizationDomain: Set[ManagedGroupRef],
```

Comment:
I find this makes the code confusing. We could have an AD object that contains a set of groups but I don't really like that either... until there are more attributes on the AD.